packages/ubuntu_desktop_installer/lib/pages/where_are_you/where_are_you_page.dart

```diff
@@ -70,7 +70,6 @@ class WhereAreYouPageState extends State<WhereAreYouPage> {
                               formatLocation(model.selectedLocation);
                         }
                         return TextFormField(
-                          autofocus: true,
```

Comment:
in offline mode, the field is initialized to the system locale and no longer left empty